pkg/devfile/types.go

```diff
@@ -0,0 +1,31 @@
+package devfile
```

Comment:
why are we copying this instead of vendoring ?

Reply:
Sorry, I've had to wait for PR https://github.com/devfile/registry-support/pull/51 to be merged before vendoring as you requested.

I've updated, PTAL!